tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -859,10 +873,10 @@ cc_library(
 )
 
 tf_cc_test(
-    name = "cudnn_conv_pad_for_tensor_cores_test",
-    srcs = ["cudnn_conv_pad_for_tensor_cores_test.cc"],
+    name = "cudnn_pad_for_convolutions_test",
```

Comment:
The new pad_for_convolution transform combining pad_for_tensor_cores and pad_for_integer_convolutions from the previous commit.